docs/source/developer/xkcd_extension_tutorial.rst

```diff
@@ -259,6 +259,13 @@ to get a reference to the command palette interface.
       ICommandPalette
     } from '@jupyterlab/apputils';
 
+We will also need to install this dependency. Run the following command in the
+repository root folder.
```

Comment:
For this and the other additions of this nature: I think it reads more smoothly to have it end in a ":".

Also, I see there is some mix in the voice of this article. There is some first person plural ("we"), but much more if it is second-person singular. I think that the latter conforms more closely to our documentation [guidelines](https://github.com/jupyterlab/jupyterlab/blob/master/docs/source/developer/documentation.rst#writing-style).

Reply:
I agree

Reply:
"We" - Sorry, still have the habit from math papers :).

Reply:
> For this and the other additions of this nature: I think it reads more smoothly to have it end in a ":".

The commands in the first part all don't have the colon. There are a few in the last part of the article that do have a colon. We should probably clean that up on another PR.